### PR TITLE
Bulk download

### LINKED ISSUE #474: Bulk Download
- [x] Include Intervals, Annual, FRA_Years files (these files are available from admin download)
- [x] Add Bulk Download button only for Global 

Screen Shot 2020-08-19 at 11 17 07

### PR DESCRIPTION
**Pull request by sorja**
Resolves #474

### CODE REVIEW

File: server/assessment/service/exportService.js [hunk 1]
```diff
@@ -20,6 +20,43 @@ const EXPORT_TYPE = {
   CSV: 'csv',
 }
 
+// This is used in Global => navigation => Bulk Download
+const exportPublicData = async () => {
```
Comment (minotogna) [inline server/assessment/service/exportService.js:24]: why duplicate this method? wouldn't it be more effective to add some condition to the existing exportData method ?

File: webapp/app/components/navigation/navigation.less [hunk 2]
```diff
@@ -14,6 +14,12 @@
   overflow-y: auto;
   height: 100%;
   padding: 8px 0;
+
+  .nav__bulk-download {
+    margin-left: 16px;
+    margin-top: 8px;
```
Comment (minotogna) [inline webapp/app/components/navigation/navigation.less:20]: margin-top even 24px

File: server/assessment/service/annual/annualYearsExporter.js [hunk 3]
```diff
@@ -3,65 +3,61 @@ const Promise = require('bluebird')
...
-    //5a, 5b
+    // 5a, 5b
     ...DisturbancesExporter.parseResultRow(disturbances, yearIdx, year),
     ...AreaAffectedByFireExporter.parseResultRow(areaAffectedByFire, yearIdx, year),
   }))
-
 }
 
-const getCsvOutput = () => {
+const getCsvOutput = (noVariableFolder) => {
```
Comment (minotogna) [inline server/assessment/service/annual/annualYearsExporter.js:51]: can we call this `includeVariableFolders` ? it's more meaningful

File: server/assessment/service/exportService.js [hunk 4]
```diff
@@ -20,62 +18,91 @@ const EXPORT_TYPE = {
   CSV: 'csv',
 }
 
-const exportData = async (user, exportType = EXPORT_TYPE.JSON) => {
-  AccessControl.checkAdminAccess(user)
-
+// if simple flag is true, only return fra years, intervals and annualoutput without subfolders
+const exportData = async (exportType = EXPORT_TYPE.JSON, simple) => {
```
Comment (minotogna) [inline server/assessment/service/exportService.js:22]: call the second param includeSubFolders?

File: server/dataExport/api.js [hunk 5]
```diff
@@ -18,4 +21,21 @@ module.exports.init = (app) => {
       Request.sendErr(res, err)
     }
   })
+
+  app.get('/export/FRA-bulk-download', async (req, res) => {
```
Comment (minotogna) [inline server/dataExport/api.js:25]: endpoint can be `/export/bulk-download`

File: webapp/app/components/navigation/navigation.js [hunk 6]
```diff
@@ -27,6 +30,13 @@ const Navigation = () => {
 
       <LinkPanEuropeanIndicators />
 
+      {isISOGlobal(countryIso) && (
```
Comment (minotogna) [inline webapp/app/components/navigation/navigation.js:33]: this should also check that assessment is fra

File: server/assessment/service/annual/annualYearsExporter.js (hunk 3 above)
Comment (sorja) [reply server/assessment/service/annual/annualYearsExporter.js:51]: yes, but I didnt want to touch the old code - I can change this

File: server/assessment/service/exportService.js (hunk 4 above)
Comment (sorja) [reply server/assessment/service/exportService.js:22]: you mean excludeSubFolders?

File: server/assessment/service/annual/annualYearsExporter.js (hunk 3 above)
Comment (sorja) [reply server/assessment/service/annual/annualYearsExporter.js:51]: It is used widely, leaving as is if ok

Comment (minotogna) [reply server/assessment/service/annual/annualYearsExporter.js:51]: but I don't understand what noVariableFolder means. can you explain ?

Comment (sorja) [reply server/assessment/service/annual/annualYearsExporter.js:51]: It is a toggle variable for excluding the subfolders in the zip file

Comment (sorja) [reply server/assessment/service/annual/annualYearsExporter.js:51]: I can always rename it to `includeVariableFolders` and set the default value to true

**Review by minotogna — APPROVED** [review]

**Pull request merged**